src/Page.php

```diff
@@ -1068,4 +1068,25 @@ public function setUserAgent(string $userAgent)
 
         return new ResponseWaiter($response);
     }
+
+    /**
+     * Disable or enable JavaScript execution for the current page.
+     *
+     * @param bool $enabled
+     *
+     * @throws CommunicationException
+     *
+     * @return ResponseWaiter
+     */
+    public function setScriptExecution(bool $enabled)
```

Comment:
```suggestion
     * @throws CommunicationException
     */
    public function setScriptExecution(bool $enabled): ResponseWaiter
```

Those param and return docblocks are unnecessary if we use the types correctly directly in the method itself. Other then that, the code looks good to me.